pkg/apis/node/v1alpha1/types.go

```diff
@@ -92,19 +92,35 @@ type Property struct {
 
 type CustomNodeResourceStatus struct {
 	// +optional
+	// +kubebuilder:deprecatedversion:warning="this field is deprecated and should be moved to Resources"
 	ResourceAllocatable *v1.ResourceList `json:"resourceAllocatable,omitempty"`
 
 	// +optional
+	// +kubebuilder:deprecatedversion:warning="this field is deprecated and should be moved to Resources"
 	ResourceCapacity *v1.ResourceList `json:"resourceCapacity,omitempty"`
 
+	Resources Resources `json:"resources"`
+
 	// +optional
-	TopologyStatus *TopologyStatus `json:"topologyStatus,omitempty"`
+	TopologyStatus []*TopologyStatus `json:"topologyStatus,omitempty"`
+
+	// TopologyPolicy indicates placement policy for scheduler or other centralized components to follow.
+	// this policy (including topology scope) is defined in topology-manager, katalyst is
+	// responsible to parse the policy, and transform to TopologyPolicy here.
+	// +kubebuilder:default:=TopologyPolicyNode
```

Comment:
maybe there is a typo: TopologyPolicyNode -> TopologyPolicyNone

Reply:
or should we set default:=none ?

Reply:
fixed